### PR TITLE
Add io.itch.domestique_baston.kindergarten_massaker

### PR DESCRIPTION
**Pull request by glevner**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  Action game: an alien lands in a kindergarten where the children quickly turn hostile.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**
https://domestique-baston.itch.io/kindergarten-massaker

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
[the remark this answers is not in the document]
Comment (glevner) [reply flathub.json:2]: Can you live with the disappointment?

[the remark this answers is not in the document]
Comment (hfiguiere) [reply flathub.json:2]: I thought godot was cross platform?

[the remark this answers is not in the document]
Comment (glevner) [reply flathub.json:2]: We have three different games out there at the moment. We **could** publish 32-bit Linux versions of them. But can we worry about that afterward, maybe?

[the remark this answers is not in the document]
Comment (hfiguiere) [reply flathub.json:2]: You can just export the pack file and have a different godot binary per arch, can't you?

[the remark this answers is not in the document]
Comment (glevner) [reply flathub.json:2]: By using the Godot base app, you mean? I tried that with our first game, but never got it to work.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply flathub.json:2]: who cares about 32-bits? We only support 64-bits architectures. aarch64 is the other one.

**Comment by bbhtt** [comment]
bot, build io.itch.domestique_baston.kindergarten_massaker

**Comment by glevner** [comment]
bot, build io.itch.domestique_baston.java_jolt

**Comment by glevner** [comment]
> Queued test build for io.itch.domestique_baston.java_jolt.

Oops, wrong app...

**Pull request closed without merging**